test/e2e/systemd_test.go

```diff
@@ -0,0 +1,65 @@
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	. "github.com/containers/libpod/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman systemd", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+	)
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanTestCreate(tempdir)
+		podmanTest.RestoreAllArtifacts()
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+		f := CurrentGinkgoTestDescription()
+		timedResult := fmt.Sprintf("Test: %s completed in %f seconds", f.TestText, f.Duration.Seconds())
+		GinkgoWriter.Write([]byte(timedResult))
+	})
+
+	It("podman start container by systemd", func() {
+		systemd_unit_file := []byte("
+[Unit]\n
+Description=redis container\n
+[Service]\n
+Restart=always\n
+ExecStart=/usr/bin/podman start -a redis\n
+ExecStop=/usr/bin/podman stop -t 10 redis\n
+KillMode=process\n
+[Install]\n
+WantedBy=multi-user.target"
+		sys_file: ioutil.WriteFile("/etc/systemd/system/redis.service", systemd_unit_file, 0644)
+		check(sys_file)
```

Comment:
Probably want a `defer os.Remove("/etc/systemd/system/redis.service")` after this to ensure the service file is cleaned up after the test

Reply:
done